epersons.md

```diff
@@ -160,6 +160,9 @@ the replace operation `[{ "op": "replace", "path": "/password", "value": "newpas
 ```
 NOTE: The new password is currently returned after an update but this could be revisited later, see [#30]((https://github.com/DSpace/Rest7Contract/issues/30))
 
+Any EPerson metadata can be modified by both the administrator and by the authenticated user.  
```

Comment:
One tiny thing, as worded, I just realized this seems to imply the "authenticated user" can modify *any* EPerson.  That's obviously not correct.  Probably should reword to something like:
`The currently authenticated user can modify their EPerson metadata.  An Administrator can modify any EPerson's metadata.`

Reply:
Thanks for the suggestion, that is indeed easier to read